packages/specs/schema/src/domain/JsonSchema.ts

```
@@ -17,14 +33,13 @@ import {JsonEntityStore} from "./JsonEntityStore.js";
 import {JsonFormatTypes} from "./JsonFormatTypes.js";
 import {JsonLazyRef} from "./JsonLazyRef.js";
 
-export interface JsonSchemaObject extends JSONSchema6, Record<string, any> {
-  type: (any | JSONSchema6TypeName) | (any | JSONSchema6TypeName)[];
-  additionalProperties?: boolean | JSONSchema6 | any;
-  propertyNames?: boolean | JSONSchema6 | any;
-  items?: (any | JSONSchema6Definition) | (any | JSONSchema6Definition)[];
+export interface JsonSchemaObject extends Omit<JSONSchema7, "type" | "additionalProperties" | "items"> {
+  type?: Type | JSONSchema7["type"] | null | (String | null | Date | Number | Object | Boolean)[];
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)